app/views/pages/crayons.html.erb

```diff
@@ -76,7 +76,7 @@
 <div class="container">
   <div class="body">
     <h2>Main Typography</h2>
-    <p>Default font is set to 16px (fs-base). It should be standard in UI. Smaller and bigger font sizes should be used carefully with respect to good visual rythm between elements.</p>
+    <p>Default font is set to 16px (fs-base). It should be standard in UI. Smaller and bigger font sizes should be used carefully with respect to good visual rhythm between elements.</p>
```

Comment:
just a thought now I'm seeing this, but maybe we should change this to say something like 

```suggestion
    <p>Default font is set to 1rem (fs-base). In standard browser settings this equates to 16px. Smaller and bigger font sizes should be used carefully with respect to good visual rhythm between elements.</p>
```

just thinking it's not totally accurate to say it's set to 16px, even though in most cases that's what 1rem ends up being

Reply:
I'll actually deprecate this page - it's been there mostly as a playground for myself and I don't think it's very useful to anyone - it's actually the opposite if we want to polish our Storybook and make it the single source of truth.

Nevertheless, that's a good catch!